### ISSUE TITLE
feature: pan / zoom / morph

### ISSUE THREAD

**Issue by surjikal** [dribnet/pixray#3]
It would be great to add a few parameters or hooks that would allow you to control how the image gen changes over time, including controls for panning / zooming.

There's a cool example on reddit:
https://www.reddit.com/r/bigsleep/comments/pjd3u6/outer_space15_d_05_bottom_of_the_ocean15_u_05/

The prompt is:
```
outer space:15_d_0.5 | bottom of the ocean:15_u_0.5 | lush foliage | skeleton gardener floating:5 | by james gurney:4 | by kinkade:10 || red ashy mountains:1_u_0.5 | retro beach:1_d_0.6 | palm trees:1_d_0.3 | clear blue sky:1_u_0.25 | by james gurney:1 | by kinkade:3 | 20megapixels:1
```

I assume some of these are parameters to control the "camera".

**Comment by dribnet** [comment]
agreed that some animation controls would be nice to add. i believe in the example you posted, the camera is hard coded and all of those arguments are being parsed into components of the text prompt. 

i'll leave this open for now and will update when i have some animation possibilities.

**Comment by surjikal** [comment]
You're correct, these are weights and the camera movement is hardcoded. The python library is available on github, but I don't want to put the link here since I go it through the author's patreon here: https://www.patreon.com/sportsracer48
